src/app/check_opam_switch/dune

```diff
@@ -0,0 +1,15 @@
+(executable
```

Comment:
throwing that out there, but it might be nice to have that live on its own repo, and published to opam, for other people to use